force-app/main/default/classes/SOM_TestDataFactory.cls

```diff
@@ -77,6 +77,7 @@ public class SOM_TestDataFactory {
         order.EffectiveDate = input.effectiveDate;
         order.Status = input.status;
         order.SalesChannelId = input.salesChannelId;
+        order.orderedDate = Date.today().addDays(-3);
```

Comment:
you should get this value from the input as in the other lines, in this case is the variable you changed when creating the input 'effectiveDate'